src/shared/config.ts

```diff
@@ -92,6 +130,34 @@ const ConvictConfig = Convict<ServiceConfig>({
       format: 'Boolean',
       default: true
     }
+  },
+  // TODO: Add documentation
+  SHARED: {
+    PEER_ENDPOINT: '0.0.0.0:4003',
+    ALS_ENDPOINT: '0.0.0.0:4002',
+    QUOTES_ENDPOINT: '0.0.0.0:3002',
+    TRANSFERS_ENDPOINT: '0.0.0.0:3000',
+    BULK_TRANSFERS_ENDPOINT: '',
+    JWS_SIGN: false,
+    JWS_SIGNING_KEY: '',
+    WSO2_AUTH: {
+      staticToken: '',
+      tokenEndpoint: '',
+      clientKey: '',
+      clientSecret: '',
+      refreshSeconds: 60
+    },
+    // TODO: Investigate proper key setup
```

Comment:
TODO

Reply:
This TODO was carried over from the `thirdparty-scheme-adapter` not sure how to resolve this

Reply:
drop these TODO comments